src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student8/BookStoreController8.java

```diff
@@ -0,0 +1,60 @@
+package com.xpanxion.java.springboot.da1.demo.controller.student8;
+
+import com.xpanxion.java.springboot.da1.demo.model.student8.Book;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class BookStoreController8 {
+
+    @Autowired
+    private DataAccess8 dataAccess;
+
+    @GetMapping("student8/api/v1/bookstores/{bookstoreId}/books")
+    public List<Book> getBooksInBookstore(@PathVariable("bookstoreId") Integer id) {
+        var bookList = dataAccess.getBooksInBookstore(id);
+        return bookList;
+    }
+
+    @Service
+    static class DataAccess8 {
+
+        // DATA
+
+        @Autowired
+        private JdbcTemplate jdbcTemplate;
+        private final String SELECT_BOOKS_BY_BOOKSTORE = """
+            select book.title, book.book_id, book.isbn, book.price, bookstore_book.quantity\n
```

Comment:
We don't need to provide a newline character here `\n`.  It can be removed on all the lines.

Reply:
updated in new commit